### DISCUSSION TITLE
Switch tag for Fluid fields

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3770] [Feature Requests]
### Suggest an idea for ExpressionEngine

I'd really like a {switch} tag for Fluid fields please.... to operate just like for Grid fields.
Thank yooouuuuu....  😃 

{fluid_field_name:switch="1|2"}

**Comment by intoeetive** [comment]
I think you can achieve similar results by using this conditional check:
```
{if fluid_field_name:count % 2 == 0}
... is even ...
{if:else}
... is odd ...
{/if}
```

**Reply by ic360-com** [reply]
Oh cool - didn't know about this function. Thanks.
